analyzer/protobuf.spicy

```diff
@@ -83,10 +82,63 @@ function parse_varint(buf: vector<uint8>) : uint64 {
     return parsed;
 }
 
+# TODO: Surely there's a better way...
+function to_bytes(u: uint64) : bytes {
```

Comment:
The necessity of this function is to work around integer-overflow exceptions that get thrown when trying to reinterpret unsigned integers as signed integers. I'm thinking it has to be user error on my part and that this isn't really necessary.